README.md

```diff
@@ -83,3 +83,11 @@ Developed by [Land Information New Zealand](https://github.com/linz)
      ```
      $ black . --check --diff
      ```
+
+To add a development-only package: `poetry add --dev PACKAGE='*'`
+
+To add a production package:
+
+1. Install the package using `poetry add --optional PACKAGE='*'`.
```

Comment:
Copy/paste error in line numbers ?

Reply:
That's just how Markdown works: Any number before the dot results in the list being rendered as a numbered list, starting at 1. It's pretty common to make all the numbers 1 rather than getting a bigger diff when adding and removing entries, and dealing with re-numbering all of them.

Reply:
ah, I didn't know. I had a feeling that it might be something like that.